src/predict_test.py

```diff
@@ -0,0 +1,145 @@
+"""
+Author: Aidan Mattrick
+
+Date: Dec 9, 2020
+
+This script takes in a trained model and runs it against the test set and returns a confusion matrix outline the results.
+
+Usage: predict_test.py -i=<input> -o=<output> [-f <filename>] [-v]
+
+Options:
+-i <input>, --input <input>                 Local processed training data csv file in directory
+-o <output>, --output <output>              Local output directory for created pngs
+-f <filename>, --filename <filename>        Add a prefix to the saved filename
+[-v]                                        Report verbose output of dataset retrieval process
+"""
+
+import os
+import re
+import sys
+from hashlib import sha1
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from sklearn import datasets
+from sklearn.compose import (
+    ColumnTransformer,
+    TransformedTargetRegressor,
+    make_column_transformer,
+)
+from sklearn.dummy import DummyClassifier, DummyRegressor
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LogisticRegression, Ridge, LinearRegression
+from sklearn.multiclass import OneVsRestClassifier
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.svm import SVC
+from sklearn.metrics import (
+    accuracy_score,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    make_scorer,
+    precision_score,
+    recall_score,
+)
+from sklearn.model_selection import (
+    GridSearchCV,
+    RandomizedSearchCV,
+    cross_validate,
+    train_test_split,
+)
+from sklearn.pipeline import Pipeline, make_pipeline
+from sklearn.preprocessing import OneHotEncoder, OrdinalEncoder, StandardScaler
+from sklearn.svm import SVC, SVR
+from sklearn.metrics import plot_confusion_matrix
+
+
+from lightgbm.sklearn import LGBMClassifier
+from catboost import CatBoostClassifier
+
+import altair as alt
+from vega_datasets import data
+
+from docopt import docopt
+from pylab import savefig
+from render_table import render_table
```

Comment:
hey there, render table is not in utils, you need to change that by 'from character_utils import*'

Reply:
OR even better: from character_utils import render_table, read_input_file

Reply:
Removed render_table as not being used!